src/SFA.DAS.AdminService.Web/Services/Gateway/GatewayCriminalComplianceChecksOrchestrator.cs

```diff
@@ -23,24 +23,28 @@ public class GatewayCriminalComplianceChecksOrchestrator : IGatewayCriminalCompl
         }
 
         public async Task<OrganisationCriminalCompliancePageViewModel> GetCriminalComplianceCheckViewModel(GetCriminalComplianceCheckRequest request)
-        {          
-            var commonDetails = await _applyApiClient.GetPageCommonDetails(request.ApplicationId, request.PageId, request.UserName);
-
-            var model = new OrganisationCriminalCompliancePageViewModel 
-            { 
-                ApplicationId = request.ApplicationId, 
-                PageId = request.PageId,
-                ApplyLegalName = commonDetails.LegalName,
-                Ukprn = commonDetails.Ukprn,
-                Status = commonDetails.Status,
-                OptionPassText = commonDetails.OptionPassText,
-                OptionFailText = commonDetails.OptionFailText,
-                OptionInProgressText = commonDetails.OptionInProgressText,
-                GatewayReviewStatus = commonDetails.GatewayReviewStatus
-            };
+        {
+            //var commonDetails = await _applyApiClient.GetPageCommonDetails(request.ApplicationId, request.PageId, request.UserName);
 
+            //var model = new OrganisationCriminalCompliancePageViewModel 
+            //{ 
+            //    ApplicationId = request.ApplicationId, 
+            //    PageId = request.PageId,
+            //    ApplyLegalName = commonDetails.LegalName,
+            //    Ukprn = commonDetails.Ukprn,
+            //    Status = commonDetails.Status,
+            //    OptionPassText = commonDetails.OptionPassText,
+            //    OptionFailText = commonDetails.OptionFailText,
+            //    OptionInProgressText = commonDetails.OptionInProgressText
+            //};
             _logger.LogInformation($"Retrieving criminal compliance details for application {request.ApplicationId} page {request.PageId}");
 
+            var model = new OrganisationCriminalCompliancePageViewModel();
+            await model.PopulatePageCommonDetails(_applyApiClient, request.ApplicationId, request.PageId, request.UserName);
+            model.Caption = RoatpGatewayConstants.Captions.OrganisationsCriminalAndComplianceChecks;
+            model.Heading = CriminalCompliancePageConfiguration.Headings[request.PageId];
+            model.NoSelectionErrorMessage = CriminalCompliancePageConfiguration.NoSelectionErrorMessages[request.PageId];
```

Comment:
Is it worth putting these 3 sets in PopulatePageCommonDetails and passing the values as parameters?  That way it'll prevent people (me! 😁) forgetting to set them.

Reply:
Now we are passing the values as parameters, but you still will need to remember to set them appropriately. :-)